docs/custom_language_detection.rst

```diff
@@ -0,0 +1,89 @@
+=========================
+Custom language detection
+=========================
+
+`dateparser` allows to customize the language detection behavior by using the ``detect_languages_function`` parameter. 
+It currently supports two language detection libraries out of the box: `fasttext <https://github.com/facebookresearch/fastText>`_ 
+and `langdetect <https://github.com/Mimino666/langdetect>`_, and allows you to implement your own custom language detection.
+
+
```

Comment:
Add Warning saying "For short strings the language detection could fail, so it's highly recommended to use this ``detect_languages_function`` along with some values in `DEFAULT_LANGUAGES`".

Reply:
Done 👍🏻 

I have added a note

Reply:
Done 👍🏻